### PR TITLE
WF-65375-Add information about HTML export option and Apply paragraph formatting

### PR DESCRIPTION
**Pull request by sharonaumao**
<table>
<tr>
<td>
Task Link
</td>
<td>
<a href="https://syncfusion.atlassian.net/browse/WF-65424">https://syncfusion.atlassian.net/browse/WF-65424</a>
<a href="https://syncfusion.atlassian.net/browse/WF-62411">https://syncfusion.atlassian.net/browse/WF-62411</a>
</td>
</tr>
<tr>
<td>
Description
</td>
<td>
1. Add information about HTML export option
2. Apply paragraph formatting
</td>
</tr>
<tr>
<td>
Solution
</td>
<td>
1. Add information about HTML export option
2. Apply paragraph formatting
</td>
</tr>
<tr>
<td>
Content review confirmation
</td>
<td>
 Done
</td>
</tr>
</table>

### CODE REVIEW

File: File-Formats/DocIO/html.md [hunk 1]
```diff
@@ -439,6 +439,8 @@ You can customize the Word to HTML conversion with the following options:
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/html.md:439]: Add about the API here

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/html.md (hunk 1 above)
Comment (sharonaumao) [reply File-Formats/DocIO/html.md:440]: @Suriya-Balamurugan -Addressed

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**